test/helpers/maintenance_tasks/tasks_helper_test.rb

```diff
@@ -91,14 +91,19 @@ class TasksHelperTest < ActionView::TestCase
         csv_file_download_path(run)
     end
 
+    test "#resolve_inclusion_value resolves inclusion validator for task attributes" do
+      assert_match "Select a value", markup("integer_dropdown_attr").squish
+
+      assert_match "Select a value", markup("boolean_dropdown_attr").squish
+
+      ["text_integer_attr", "text_integer_attr2", "text_integer_attr3"].each do |text_integer_attr|
+        refute_match "Select a value", markup(text_integer_attr).squish
+      end
+    end
```

Comment:
LMK if this unit test feels redundant now -- since it should fail on the [system test](https://github.com/Shopify/maintenance_tasks/blob/2fda52bf088a999009902a81c9207c700e88f922/test/system/maintenance_tasks/tasks_test.rb), and I can also get rid of this [minor refactor ](https://github.com/Shopify/maintenance_tasks/pull/962/commits/2fda52bf088a999009902a81c9207c700e88f922) to keep it short